frame/vesting/src/mock.rs

```diff
@@ -101,11 +101,12 @@ impl Config for Test {
 pub struct ExtBuilder {
 	existential_deposit: u64,
 	vesting_genesis_config: Option<Vec<(u64, u64, u64, u64)>>,
+	schedules: Vec<(u64, VestingInfo<u64, u64>)>,
 }
 
 impl Default for ExtBuilder {
 	fn default() -> Self {
-		Self { existential_deposit: 1, vesting_genesis_config: None }
+		Self { existential_deposit: 1, vesting_genesis_config: None, schedules: vec![] }
```

Comment:
Why did you add another field? Can you not just put it into `vesting_genesis_config`?  
I would rather have one field, so you can either put it all in `vesting_genesis_config` or in `schedules` which is probably cleaner since it has a type instead of a tuple.

Reply:
I don't think I can put them into `vesting_genesis_config`. When I did some tests checking `usable_balance` on accounts with multiple schedules failed, since the genesis build function would only update the vesting lock with the details of the last schedule of that account in the iteration.

Then when I tried to make them all schedules instead and call `vested_transfer` on them, the `generates_multiple_schedules_from_genesis_config` and `multiple_schedules_from_genesis_config_errors` tests fail.